README.md

```diff
@@ -31,16 +31,18 @@ I wanted to write down my learnings and make an approachable guide for people wh
 ------------------
 
 ### Guide
+
 1. [The CPU](#the-cpu)
-1. [Electricity and the physical world](#electricity-and-the-physical-world)
-    - [Saving data](#saving-data)
-1. [Writing Code](#writing-code)
-1. [The Math Section](#the-math-section)
-    - [Binary](#binary)
-    - [Boolean Logic](#boolean-logic)
-    - [Hexadecimal](#hexadecimal)
-1. [Resources](#resources)
-1. [Contributors](#contributors)
+   - [Electricity and the physical world](#electricity-and-the-physical-world)
+   - [Saving data](#saving-data)
+2. [Communicating with the CPU](#communicating-with-the-cpu)
```

Comment:
If we keep these at `1.`s, then the markdown will auto increment, which is nice if we're moving around sections!

Reply:
ooo that's cool, thanks! will keep in mind.